src/cc/tests/access/CMakeLists.txt

```diff
@@ -1,21 +0,0 @@
-# Licensed under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License. You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-# implied. See the License for the specific language governing
-# permissions and limitations under the License.
-
-set(sources
-
-)
-
-foreach(source ${sources})
-    list(APPEND test_sources ${CMAKE_CURRENT_LIST_DIR}/${source})
```

Comment:
Yeah, I was originally going to have sub-builds and automatically scan directory contents to create the build tree but I couldn't get scoping working properly in cmake so we'll just do it this way for now. It's less verbose anyway.